CONTRIBUTING.md

```diff
@@ -32,57 +34,22 @@ Confirm a successful set up by running the following:
 docker ps
 ```
 
-:wrench: Note: Consider running the following to see the container logs real time:
-
-```bash
-docker logs <your-container-id> -f
-```
-
 ### 3. Clone and compile `pg_vectorize` and extension dependencies
 
-:wrench: When progressing through these steps, refer to the following for troubleshooting:
-
-```bash
-cat ~/.pgrx/15.log
-```
```

Comment:
we should leave "how to get logs from pgrx" somewhere in the contrib guide since its not super straight forward where they are located

Reply:
Adding the troubleshooting section at line 169